src/Util/RequestAttributesExtractor.php

```diff
@@ -39,9 +39,14 @@ private function __construct()
      */
     public static function extractAttributes(Request $request)
     {
-        $result = ['resource_class' => $request->attributes->get('_api_resource_class')];
+        return self::extractAttributesFromParameters($request->attributes->all());
+    }
+
+    public static function extractAttributesFromParameters(array $attributes)
```

Comment:
I would introduce a new class, as it's not tight with the `Request` nor Symfony anymore.

Reply:
Also, please copy/adapt the PHPDoc.